With dev-libs/libxml2-2.11, xi:include errors out if it can't find a file, so add an empty fallback.
Used to work with dev-libs/libxml2-2.10.

See also: https://bugs.gentoo.org/906309

--- a/doc/manual/en_US/user_Frontends.xml
+++ b/doc/manual/en_US/user_Frontends.xml
@@ -1205,6 +1205,6 @@
 
   </sect1>
 
-  <xi:include href="user_man_VBoxHeadless.xml" xpointer="element(/1)" xmlns:xi="http://www.w3.org/2001/XInclude" />
+  <xi:include href="user_man_VBoxHeadless.xml" xpointer="element(/1)" xmlns:xi="http://www.w3.org/2001/XInclude" ><xi:fallback/></xi:include>
 
 </chapter>
--- b/doc/manual/en_US/user_VBoxManage.xml
+++ a/doc/manual/en_US/user_VBoxManage.xml
@@ -8975,6 +8975,6 @@
 <!-- TODO: Figure out how we can handle other manpages. The xml is bolted to
              sect1, so it's not possible to have them "in place" -->
 
-  <xi:include href="user_man_vboximg-mount.xml" xpointer="element(/1)" xmlns:xi="http://www.w3.org/2001/XInclude" />
+  <xi:include href="user_man_vboximg-mount.xml" xpointer="element(/1)" xmlns:xi="http://www.w3.org/2001/XInclude" ><xi:fallback/></xi:include>
 
 </chapter>
